layers/update-layers.py

```diff
@@ -105,7 +99,7 @@ def update_layer(layerfile, replace=False):
             technique["techniqueID"] = newID
     
     # set the version to current
-    layer["version"] = "3.0"
+    layer["version"] = "4.0"
```

Comment:
This script should also remove `layer["filters"]["stages"]` if it is present in the input json.